core/model/user.go

```diff
@@ -112,3 +112,11 @@ type DeletedMembership struct {
 	AccountID string                  `json:"account_id"`
 	Context   *map[string]interface{} `json:"context,omitempty"`
 }
+
+// UserDataResponse represents a user data
+type UserDataResponse struct {
```

Comment:
This does not correspond with the desired response which is:
```
{
    "received_notifications": [
        {
            //the same message structure as in the get messages API. //from messages_recipients
        }
    ],
    "scheduled_notifications_for_me":[
        {
            //the same message structure as in the get messages API. //from gueue_data.
        }
    ],
    "my_account": {
        //the same structure as get user API - //from users.
    }
}
```

Reply:
This is not fixed:
_"received_notifications": [
        {
            //the same message structure as in the get messages API. //from messages_recipients
        }
    ],
    "scheduled_notifications_for_me":[
        {
            //the same message structure as in the get messages API. //from gueue_data.
        }
    ],_

Do not return MessageRecipeint and QueueItem in the response